code/game/objects/items/weapons/shields.dm

```diff
@@ -21,6 +21,22 @@
 	origin_tech = "materials=2"
 	block_chance = list(melee = 60, bullet = 50, laser = 50, energy = 50) //Skeleton-exclusive atm, sooo
 
+/obj/item/weapon/shield/roman/replica
```

Comment:
A lot of the definitions you have here (e.g. name, desc, etc.) are the same as the parent type (`/obj/item/weapon/shield/roman`), so you don't need to define them again as it'll inherit the parent's values.

Also perhaps we should name it "replicate roman shield"?